src/main/java/com/musala/atmosphere/server/pool/PoolManager.java

```diff
@@ -37,31 +29,26 @@
 import com.musala.atmosphere.server.eventservice.event.device.publish.DevicePublishedEvent;
 import com.musala.atmosphere.server.eventservice.event.device.publish.DeviceUnpublishedEvent;
 import com.musala.atmosphere.server.eventservice.subscriber.Subscriber;
-import com.musala.atmosphere.server.registrymanager.RemoteObjectRegistryManager;
 
 /**
- * Class which is responsible for iterating with the devices in the pool. It also sending events to
- * {@link RemoteObjectRegistryManager} when e device is add to the pool or removed.
+ * Class which is responsible for iterating with the devices in the pool.
  *
  * @author yavor.stankov
  *
  */
-public class PoolManager extends UnicastRemoteObject implements IClientBuilder, Subscriber {
-    private static final long serialVersionUID = -5077918124351182199L;
-
-    private static final String DEVICE_RMI_ID_FORMAT = "%s_%s";
+public class PoolManager implements Subscriber {
+    private static final String DEVICE_ID_FORMAT = "%s_%s";
```

Comment:
I didn't find this comment but I think I understood this remark. `buildDeviceIdentifier(...)` can be used statically: `Poolmanager.buildDeviceIdentifier(..)` and I don't need of something like `DEVICE_ID_FORMATTER` in the `ServerManager`.